install.sh

```diff
@@ -308,18 +327,12 @@ sudo ldconfig &> /tmp/crew_ldconfig || true
 echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
 
-echo_out "Set up and synchronize local package repo..."
-
 echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
 
-# Install activesupport gem for ruby
-echo_info 'Installing essential ruby gems.'
-gem update -N --system
-gem install -N activesupport --conservative
-gem install -N concurrent-ruby --conservative
-
-# Since we downloaded the package repo, just update package compatibility information.
-crew update compatible
+echo_info 'Installing essential ruby gems...\n'
```

Comment:
Does `\n` inside single quotes print the literal and not a linefeed?

Reply:
Fixed.